index.html

```diff
@@ -0,0 +1,121 @@
+                </p>
+            </div>
+            <div class="program_card">
+                <img src="./assets/images/program_icon_02 1.svg" alt="CC Exhibition">
+                <h5 class="main-text-color">Exhibition</h5>
+                <p>
+                    The Africa Day concert is organised in African Union’s Division of Culture and Sports. Celebrated annually,
+                </p>
+            </div>
+            <div class="program_card">
+                <img src="./assets/images/program_icon_03 1.svg" alt="Forum">
+                <h5 class="main-text-color">Day servant</h5>
+                <p>
+                    Speaking ahead of their performance, Santrofi, who are the headlining acts
+                </p>
+            </div>
+            <div class="program_card">
+                <img src="./assets/images/program_icon_04 1.svg" alt="Workshop">
+                <h5 class="main-text-color">Workshop</h5>
+                <p>
+                    Try creating your own work using open source license rather than just watching at it.
+                </p>
+            </div>
+            <div class="program_card">
+                <img src="./assets/images/program_icon_05 1.svg" alt="CC Ignite">
+                <h5 class="main-text-color">CC Ignite</h5>
+                <p>
+                    We envision a sector that supports and promotes local talent, encourages innovation and experimentation,
+                </p>
+            </div>
+        </section>
+            <a class="more-program-desktop" href="#">SEE THE WHOLE PROGRAM</a>
+            <button class="more-program-mobile main-bg-color">Join Africa Day 2023 Concert </button>
+    </article>
+</section>
+<!-- add the speaker or the particpant person -->
+
+<!-- partner -->
+<section id="partners" class="partners">
+    <h2 class="title text-center">Partners</h2>
+    <div class="underline"></div>
+    <div class="group2">
+      <div class="partners-logo logos-img">
+        <a href="https://www.mozilla.org/en-US/firefox/"><img src="./assets/images/partners-images/mozilla 1.svg" alt="mozilla"></a>
+        <a href="https://www.google.com/"><img src="./assets/images/partners-images/googlekorea 1.svg" alt="google"></a>
+        <a href="https://www.naver.com/"><img src="./assets/images/partners-images/naver 1.svg" alt="naver"></a>
+        <a href="http://www.kakaocorp.com/"><img src="./assets/images/partners-images/daumkakaologo 1.svg" alt="daumkakao"></a>
+        <a href="https://www.airbnb.com/"><img src="./assets/images/partners-images/airbnb 1.svg" alt="iairbnb"></a>
+      </div>
+    </div>
+  </section>
+<footer id="footer" class="footer home-phone">
+    <section class="all-rights">
+       <div id="all-rights-logo-mobile"><img src="./assets/icons/logo.png" alt="cc korea icon"></div>
+       <div id="all-rights-info">
+        <div id="all-rights-intro"><p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+        </div>
+    </section>
```

Comment:
- [ ]  According to the design guideline the footer should contain a logo aligned to the left and text aligned to the right and the entire content center aligned. So please make sure to implement that for your design to look as expected based on the provided one 
- your design to be fixed 👇 
<img width="1330" alt="Screen Shot 2023-06-30 at 23 36 03" src="https://github.com/yusufyusie/Africa-Day-2023-Concert/assets/60146030/e438b930-1a7b-4924-95b8-9ff435211088">